src/kubernetes.cc

```diff
@@ -167,10 +167,7 @@ json::value KubernetesReader::ComputePodAssociations(const json::Object* pod)
   json::value instance_resource =
```

Comment:
I see.

[Optional] Perhaps move the reference to instance_resource lower then? This is quite far from the use case, so harder to see how / if it is used.

Reply:
It's already at the lowest point it can be without being out-of-order with `controllers` and `node_name`. But I'm changing this in #148 anyway, so we'll see how that works out.